images/base/scripts/common/setup.sh

```diff
@@ -90,9 +109,14 @@ apt-get install -y build-essential libtool
 #./configure
 ./configure --prefix=/usr
 
-make
-make check
-make install
+if [ x$MACHINE = xs390x ]
+then
+    echo FIXME: protobufs wont compile on 390, missing atomic call
```

Comment:
IIUC, the only implication of this is that we cannot invoke the protoc compiler on Z (today).